pyproject.toml

```diff
@@ -0,0 +1,37 @@
+# License and classifier list:
+# https://pypi.org/pypi?%3Aaction=list_classifiers
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    {name = "NASA EOSDIS Harmony Team", email = "christopher.d.durbin@nasa.gov"}
+]
+classifiers = [
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Operating System :: OS Independent",
+]
+description = "A library for Python-based Harmony services to parse incoming message, fetch data, stage data, and call back to Harmony."
+dynamic = ["dependencies", "optional-dependencies", "version"]
+license = { text = "License :: OSI Approved :: Apache Software License" }
+name = "harmony-service-lib"
+readme = "README.md"
```

Comment:
This replaces the `long_name` attribute from `setup.py`, and the `.md` extension is automatically used to determine that GitHub flavoured markdown is how things are rendered. (See [here](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#readme))